### DISCUSSION TITLE
Auth minimize on startup

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#1352] [Enhancements]
Would be nice if the auth app could start minimized in tray instead of poping up every boot.

**Comment by vishnukvmd** [comment]
FYI: Since the startup times are small, we'll be changing the implementation to not run in the background (or tray) in the next update.

**Reply by ghost** [reply]
Uhm, okay, but I'm not a fan of starting apps. I'm used to have things in system tray.

**Reply by gitmeED331** [reply]
That is a terrible idea, especially for those on non-traditional desktops (such Hyprland using Quickshell or Aylur's GTK Shell). I do not have a start menu, I do not a quickaccess app bar, and I am not going to create one (it would completely wreck my design). Every app I need for that are done via my system tray widget.

What is needed is for the app to be able to be "start-in-tray" and "close-to-tray" so that the window isn't so intrusive and showcasing all your accounts (security risk). 

Please reverse course on this!

**Reply by WhiteX** [reply]
This would be very useful. Maybe adding the option to start in the system stray in the settings if you don't wish to alter the default behaviour.

**Comment by StewartPolsky** [comment]
I would absolutely love to be able to minimize to tray and be able to see my OTP from the tray alone

**Comment by levantail** [comment]
Are there still any plans to implement this? Would be really nice to have the app starting directly in the tray instead of popping out in the middle of the screen each time on OS boot up

**Comment by dreadbalaur** [comment]
Any final word on this? I would love to have it start in background.
